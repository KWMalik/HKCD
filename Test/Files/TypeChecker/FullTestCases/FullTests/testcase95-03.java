class MyClass {
	public static void main(String[] args) {
		System.out.println(1);
	}
}
class SomeClass {
	int a;
	int b;
	Type a;
}