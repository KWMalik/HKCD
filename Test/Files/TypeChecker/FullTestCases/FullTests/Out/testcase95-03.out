Type error on line 6: A field named a is defined more than once in SomeClass.
Type error on line 8: A field named a is defined more than once in SomeClass.
Type error on line 8: A field named a is initialized with an uninitialized type Type.